### PR TITLE
Nations Update

### PR DESCRIPTION
**Pull request by Ty-Omaha**
**What does this PR do:**
Makes nations set epsilon security level

🆑 
fix: Nations mode sets the security level to epsilon
/ 🆑

### CODE REVIEW

File: code/game/gamemodes/nations/nations.dm [hunk 1]
```diff
@@ -16,9 +16,8 @@ datum/game_mode/nations
 		set_ai()
 		assign_leaders()
 //		remove_access()
-		for(var/mob/M in GLOB.player_list)
-			if(!istype(M,/mob/new_player))
-				M << sound('sound/effects/purge_siren.ogg')
+		set_security_level("epsilon")
```
Comment (Fox-McCloud) [inline code/game/gamemodes/nations/nations.dm]: have this call `del(world)`.

Comment (KasparoVy) [reply code/game/gamemodes/nations/nations.dm]: Should make some other even more nasty security level that's really just a wrapper for a force-restart of the server with no OOC/chat notification. Literally just ending reality.

**Comment by taukausanake** [comment]
That moment when you think Nations is still a thing but then you realize this is a double April Fools 😞

File: code/game/gamemodes/nations/nations.dm (hunk 1 above)
Comment (KasparoVy) [reply code/game/gamemodes/nations/nations.dm]: See, that's the real problem. I think people are actually enjoying the game.

**Pull request closed without merging**